tests/jballerina-unit-test/src/test/resources/test-src/query/simple-query-with-defined-type.bal

```diff
@@ -368,6 +392,29 @@ public function testQueryStreamWithError() {
     panic error("Expeted error, found: " + (typeof oddNumberList).toString());
 }
 
+public function testQueryStreamWithDifferentCompletionType() {
+    NumberGeneratorWithError numGen1 = new;
+    stream<int, error?> numberStream1 = new (numGen1);
+
+    NumberGeneratorWithError2 numGen2 = new;
+    stream<int, error> numberStream2 = new (numGen2);
+
+    NumberGeneratorWithCustomError numGen3 = new;
+    stream<int, ErrorR1?> numberStream3 = new (numGen3);
+
+    int[]|error oddNumberList1 = from int num in numberStream1
```

Comment:
Do the assertions for the results as well

Reply:
Updated 03065346a7a3b551105a676a36c7cba7ad9450d2